experiments/spbu_author/README.md

```
@@ -0,0 +1,124 @@
+
+### Ячейка 1 — Импорт библиотек
+Подключает `pandas`, `matplotlib`, `litstudy`, `Counter`, `numpy`.
+
+### Ячейка 2 — Загрузка данных
+
+Определяет список из 8 преподавателей и соответствующих CSV-файлов. Каждый CSV читается **отдельно** через `pd.read_csv()` в словарь `dfs_per_author` — это позволяет корректно считать число публикаций каждого автора без потерь.
+
+Затем каждый файл загружается через `litstudy.load_csv()` с явным указанием полей:
+
+
+    litstudy.load_csv(f,
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)